src/views/shortlist/nav.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Short List View Nav Template
+ * This file loads the short list view navigation.
+ *
+ * Override this template in your own theme by creating a file at [your-theme]/tribe-events/list/nav.php
+ *
+ * @package TribeEventsCalendar
+ *
+ */
+global $wp_query;
+
+$events_label_plural = tribe_get_event_label_plural();
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+} ?>
+
+<h3 class="tribe-events-visuallyhidden"><?php echo esc_html( sprintf( esc_html__( '%s Short List Navigation', 'the-events-calendar' ), $events_label_plural ) ); ?></h3>
+<ul class="tribe-events-sub-nav">
+	<!-- Left Navigation -->
+
+	<?php if ( tribe_has_previous_event() ) : ?>
+		<li class="<?php echo esc_attr( tribe_left_navigation_classes() ); ?>">
+			<a href="<?php echo esc_url( tribe_get_listview_prev_link() ); ?>" rel="prev"><?php printf( '<span>&laquo </span> ' . esc_html__( 'Previous %s', 'the-events-calendar' ), $events_label_plural ); ?></a>
```

Comment:
The `tribe_get_listview_prev_link()` and `tribe_get_listview_next_link()` functions return links to the list view rather than the shortlist view. That doesn't necessarily need to be fixed as part of this trial PR, but it is something to note before merging.